SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -239,6 +246,24 @@ - (void)sdl_applicationDidEnterBackground:(NSNotification *)notification {
 
 #pragma mark - Stream Lifecycle
 
+#pragma mark SDLTransportType Protocol
+
+/**
+ *  Sends data to Core
+ *
+ *  @param data The data to be sent to Core
+ */
+- (void)sendData:(NSData *)data {
+    if (self.dataSession.session == nil || !self.dataSession.session.accessory.connected) {
```

Comment:
Can we just add a `connected` bool to the `SDLIAPDataSession` object instead of having to dig down into the accessory here?

Reply:
or would that be the `isSessionInProgress` bool?

Reply:
Fixed